.storybook/main.ts

```diff
@@ -1,25 +1,25 @@
-import { StorybookConfig } from '@storybook/react-webpack5';
-
-const config: StorybookConfig =  {
+import type { StorybookConfig } from "@storybook/react-vite";
+const config: StorybookConfig = {
   stories: [
-    "../stories/**/*.stories.js",
+    "../src/stories/**/*.stories.jsx",
+  ],
+  addons: [
+    "@storybook/addon-docs",
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "./local-preset.js",
   ],
-
-  addons: ["../preset.js", "@storybook/addon-essentials", "@storybook/components", "@storybook/manager-api", "@storybook/preview-api"],
-
   framework: {
-    name: "@storybook/react-webpack5",
-    options: {}
+    name: "@storybook/react-vite",
```

Comment:
Sadly I can't find a trace of it any more, but I believe Webpack frameworks might eventually be deprecated in Storybook 10. Might as well switch to the new industry "standard" bundler.